src/app/settings/open-sources/open-sources.page.html

```diff
@@ -0,0 +1,28 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-back-button></ion-back-button>
+    </ion-buttons>
+    <ion-title>Open-source licences</ion-title>
+  </ion-toolbar>
+</ion-header>
+<ion-content>
+  <ion-card *ngFor="let l of licensesArray">
+    <ion-card-header class="header-flex">
+      <b>{{l.name}}</b>
+      <p>{{l.publisher}}</p>
+    </ion-card-header>
+
+    <ion-card-content>
+      License:
+      <ion-chip>
+        <ion-label>{{l.licenses}}</ion-label>
+      </ion-chip>
+      <br />
+      <div class="link-flex">
+        <ion-icon name="open-outline"></ion-icon>
+        <a *ngIf="l.repository" [href]="l.repository">{{l.repository}}</a>
```

Comment:
Intuitively I'd put `*ngIf="l.repository"` into the parent `div`– otherwise an undefined `l.repository` would result in an `div` containing just an icon, right?

Reply:
You are right! Resolved in 5cf7569